internal/pkg/scorecard/scorecard.go

```diff
@@ -310,7 +314,59 @@ func runTests() ([]TestSuite, error) {
 	if err != nil {
 		return nil, fmt.Errorf("failed to merge test suite results: %v", err)
 	}
-	return suites, nil
+	for _, suite := range suites {
+		// convert to ScorecardOutput format
+		// will add log when basic and olm tests are separated into plugins
+		pluginResults = append(pluginResults, TestSuitesToScorecardOutput([]TestSuite{suite}, ""))
+	}
+	// Run plugins
+	pluginDir := viper.GetString(PluginDirOpt)
+	if dir, err := os.Stat(pluginDir); err != nil || !dir.IsDir() {
+		log.Warnf("Plugin directory not found; skipping plugin tests")
+	} else {
+		if err := os.Chdir(pluginDir); err != nil {
+			return nil, fmt.Errorf("failed to chdir into scorecard plugin directory")
+		}
+		// executable files must be in "bin" subdirectory
+		files, err := ioutil.ReadDir("bin")
+		if err != nil {
+			return nil, fmt.Errorf("failed to list files in %s/bin", pluginDir)
+		}
+		for _, file := range files {
+			cmd := exec.Command("./bin/" + file.Name())
+			stdout := &bytes.Buffer{}
+			cmd.Stdout = stdout
+			stderr := &bytes.Buffer{}
+			cmd.Stderr = stderr
+			err := cmd.Run()
+			if err != nil {
+				pluginResults = append(pluginResults, failedPlugin(
+					fmt.Sprintf("Failed Plugin: %s", file.Name()),
+					fmt.Sprintf("Plugin with file name `%s` failed", file.Name()),
+					fmt.Sprintf("%s:\nStdout: %s\nStderr: %s", err, string(stdout.Bytes()), string(stderr.Bytes()))))
```

Comment:
Nit: A little difficult to parse the args being passed to `failedPlugin()`. WDYT about naming the arguments before passing them in:
```Go
name := fmt.Sprintf("Failed Plugin: %s", file.Name())
description := fmt.Sprintf("Plugin with file name `%s` failed", file.Name())
log := fmt.Sprintf("%s:\nStdout: %s\nStderr: %s", err, string(stdout.Bytes()), string(stderr.Bytes()))
pluginResults = append(pluginResults, failedPlugin(name, description, log))
```

Reply:
Same for Lines 355-358